coal_petroleum/demo/purchase_order_post.xml

```diff
@@ -203,15 +203,6 @@
                 obj().env.ref('coal_petroleum.purchase_order_6')
             ).invoice_ids.ids
         "/>
-        <value eval="{'invoice_date': DateTime.today().date()}"/>
```

Comment:
If you remove that, the function becomes useless. You can therefore remove the whole `<function> `